docs/sources/_index.md

```diff
@@ -8,7 +8,7 @@ hero:
   width: 110
   image: https://raw.githubusercontent.com/grafana/grafana-infinity-datasource/main/src/img/icon.svg
   height: 110
-  description: The Infinity data source plugin allows you to query and visualize data from JSON / CSV / GraphQL / XML / HTML endpoints.
+  description: The Infinity data source plugin allows you to query and visualize data from JSON/CSV/GraphQL/XML/HTML endpoints.
```

Comment:
Probably should use a comma separated list? https://developers.google.com/style/slashes#slashes-with-alternatives